js/main.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-undef */
+var $business = document.querySelector('#restaurants')
+
+function renderRecomRestaurant(recom) {
```

Comment:
rename this to just`displayRestaurants(restaurants, $container)`

Not that you should specify the container to render it in. You want to try as much as possible to not access variables outside of this function.